internal/commands/build_test.go

```diff
@@ -136,32 +134,9 @@ func testBuildCommand(t *testing.T, when spec.G, it spec.S) {
 			})
 		})
 
-		when("--no-pull", func() {
-			it("sets pull-policy=never and logs warning", func() {
-				mockClient.EXPECT().
-					Build(gomock.Any(), EqBuildOptionsWithPullPolicy(pubcfg.PullNever)).
-					Return(nil)
-
-				command.SetArgs([]string{"image", "--builder", "my-builder", "--no-pull"})
-				h.AssertNil(t, command.Execute())
-				h.AssertContains(t, outBuf.String(), "Warning: Flag --no-pull has been deprecated")
-			})
```

Comment:
It looks like we aren't testing any successful paths for setting `pull-policy never` at this point. Would you be able to port this test to the `when("--pull-policy...` block, replacing the `--no-pull` here with `--pull-policy never` and removing the Warning assertion (and keeping the `EqBuildOptionsWithPullPolicy()` func)?

Reply:
done